mainmatter.md

```diff
@@ -150,40 +151,12 @@ Here is an example:
             "access":[  
                "read"
             ]
```

Comment:
looks incomplete

Reply:
This section is missing a `type` field, but it's an example of how to put a new data type field in place so I didn't think that was important. I've added the type field to the example.

Reply:
Oh, I think I see what got removed now, added back.